.github/workflows/deploy_release.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  deploy_pypi:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2.0.1
```

Comment:
Is the `.0.1` important? the `test_and_deploy.yml` uses `v2`.

Reply:
no, this is just the latest version, I tried it during debugging.I don't think it matters much. In the end we will most likely be using https://dependabot.com/github-actions/ to keep this kind of stuff up-to-date.

Reply:
ok, thanks for confirming.